README.md

```diff
@@ -157,7 +157,9 @@ installation instructions given for Zonemaster have been followed. A test of the
 domains `ed25519.nl` and `superdns.nl` will reveal if the Zonemaster
 installation has the support or not for algorithms 15 and 16, respectively.
 
-All supported OSs, except CentOS Linux 7, support algorithms 15 and 16.
+All supported OSs, except CentOS Linux 7, support algorithms 15 and 16 out of the
+box. To get the support in CentOS Linux 7 a newer version of OpenSSL has to be
+installed and Zonemaster-LDNS has to be instlled following special instructions.
```

Comment:
typo: "instlled" -> "installed"
And see https://github.com/zonemaster/zonemaster-engine/pull/1080 (this is about documenting how to install Zonemaster on CentOS 7 with support for algos 15 and 16).

Reply:
Fixed. I have updated the information on CentOS Linux 7 and algo 15/16.